### ISSUE TITLE
Random grey item in dropdown list.

### ISSUE THREAD

**Issue by Kiarokh** [Lundalogik/lime-elements#799]
![image](https://user-images.githubusercontent.com/35954987/83731145-4d20b000-a64a-11ea-9adf-0e651df04a59.png)

The greyed-out items is the one that has `focus` state. The `focus` state is a "highlight effect" which is used by Material Design to indicate that the user can navigate with keyboard's arrow keys ( ↑ & ↓ ) in the list. It's an `opacity: 0.12;" style applied to the `:before` pseudo element.

### So what's the problem then?
The problem is that sometimes when you open the list, the `focus` state is applied to a random item in the list (not the first one which is the default). 

### How to regenerate
1- Go to [ + ] menu, and add a deal.
2- Click the status dropdown and switch the status to "Rejection"
3- Then click the dropdown again and you'll most probably see that the second item in the list has got focused.

#### Bonus bug
After step 3 mentioned above, try to use the ↑Up arrow key to go to the first item. You will notice that you must press the ↑ twice to be able to move up.

**Comment by adrianschmidt** [comment]
I think the so called "bonus bug" is much more serious than what it seems from this PR. The keyboard navigation of the dropdown in limel-select seems to pretty much consistently break when an item has already been selected. _How_ it breaks seems somewhat inconsistent though. Sometimes one or more options can't be reached. Other times the focus can't be moved to any other option at all.

I think these two symptoms are caused by the same underlying issue, but the menu being inaccessible to keyboard navigation is significantly more serious than the relatively small visual problem that the wrong item gets a grey background… 😞

@Kiarokh Should we rename and update this issue, or should we create a new one for the "bonus bug"?

**Comment by Kiarokh** [comment]
jeez. did I make this issue? 😅 no remember 

Actually we have started removing those ripple divs and their belongings from some components. I think this issue could turn to one of those tasks, where we remove the ripple, which may fix the visual defect too. But I'm not 100% sure before testing. Therefore I am not sure whether "these two symptoms are caused by the same underlying issue". We should investigate this in the same time, but I have a feeling that the bonus bug needs another PR and issue. We should look into it closers, maybe during the cooldown

**Comment by Kiarokh** [comment]
I think this visual effect is no more... so closing the issue.
but the bug is related to this I think : https://github.com/Lundalogik/lime-elements/issues/1535

**Comment by Kiarokh** [comment]
no, it's still there. I open it again, but found a solution to get rid of these visual bugs
